src/topics/group-generator/group-generator.api.ts

```diff
@@ -30,6 +30,7 @@ const routes = async (api: Api) => {
           api.groupGenerators.generators[groupGenerator.name]
             .generationFrequency,
         generationTimestamp: groupGenerator.timestamp,
+        lastGenerationDuration: groupGenerator.lastGenerationDuration,
```

Comment:
What happen for old generation? It's just null?

Reply:
Nice catch! I'll set a null value by default